datasets/HMRC_Trade/main.py

```diff
@@ -6,338 +6,132 @@
 #       extension: .py
 #       format_name: light
 #       format_version: '1.5'
-#       jupytext_version: 1.4.2
+#       jupytext_version: 1.7.1
 #   kernelspec:
 #     display_name: Python 3
 #     language: python
 #     name: python3
 # ---
 
-# +
-#idbrs = sorted(
-#    [dist for dist in scraper.distributions if dist.title.startswith('IDBR')],
-#    key=lambda d: d.title, reverse=True)
-#idbr = idbrs[0]
-#display(idbr.title)
-#yr = idbr.title[-4:]
-#tabs = {tab.name: tab for tab in idbr.as_databaker()}
-#tabs.keys()
-#print('Year: ' + yr)
-
 # +
 from gssutils import *
 import json 
 import requests
+import pandas as pd
 
-info = json.load(open('info.json')) 
-landingPage = info['landingPage2']  
+info = json.load(open('info.json'))
 
-scraper = Scraper(landingPage)
+scraper = Scraper(seed = 'info.json')
 scraper
+
+trace = TransformTrace()
+cubes = Cubes('info.json')
 # -
 
-yr = scraper.title.index("20")
-yr = scraper.title[yr:yr+4]
-yr
+scraper.select_dataset(title = lambda x: x.endswith('data tables'), latest = True)
+scraper
+scraper.dataset.family = info["families"]
 
-myfile = scraper.distributions[0].as_pandas(sheet_name=None)
+dist = scraper.distributions[1].downloadURL
```

Comment:
change selecting distirbution by title name and not index. e.g. :
`dist = scraper.distribution(title = lambda t : 'data tables' in t).downloadURL`

Reply:
Made changes to get the latest distribution